### PR TITLE
Models creation

### PR DESCRIPTION
**Pull request by emmanuelkamala**

### CODE REVIEW

File: config/master.key [hunk 1]
```diff
@@ -0,0 +1 @@
+bdcf59fcc4417e7e61bd6e2252d8babd
```
Comment (mosaaleb) [inline config/master.key]: - Guys you should never push your master.key file into source control. Pleas add it to `.gitignore` file. [Read more here](https://blog.capsens.eu/rails-5-2-imperfect-credentials-4ef03934aea4)

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
```
Comment (mosaaleb) [inline app/models/post.rb]: - [ ] validations should be present for body and title

File: app/models/comment.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
```
Comment (mosaaleb) [inline app/models/comment.rb]: - [ ] Validation for comment text should be present.

**Comment by emmanuelkamala** [comment]
@bellom @mosaaleb Thank you for the clarification. We will take note of that.

**Pull request merged**